packages/replay-internal/src/coreHandlers/performanceObserver.ts

```diff
@@ -29,6 +39,15 @@ export function setupPerformanceObserver(replay: ReplayContainer): () => void {
     addLcpInstrumentationHandler(({ metric }) => {
       replay.replayPerformanceEntries.push(getLargestContentfulPaint(metric));
     }),
+    addClsInstrumentationHandler(({ metric }) => {
+      replay.replayPerformanceEntries.push(getCumulativeLayoutShift(metric));
+    }),
+    addFidInstrumentationHandler(({ metric }) => {
+      replay.replayPerformanceEntries.push(getFirstInputDelay(metric));
+    }),
+    addInpInstrumentationHandler(({ metric }) => {
+      replay.replayPerformanceEntries.push(getInteractionToNextPaint(metric));
+    }),
```

Comment:
We could make a handler creator here:

```javascript
function webVitalHandler(getter: X): Y {
  return ({ metric }) => void replay.replayPerformanceEntries.push(getter(metric));
}
```

Then


```suggestion
    addInpInstrumentationHandler(webVitalHandler(getInteractionToNextPaint)),
```